targets/csharp/make.js

```diff
@@ -337,6 +338,32 @@ function getRequestActions(tabbing, apiCall, isInstance) {
     return "";
 }
 
+function getCustomApiLogic(tabbing, apiCall) {
+    if (apiCall.name === "ExecuteFunction")
+    {
+        return "\n" + tabbing + "string localApiServerString = PlayFabSettings.LocalApiServer;\n"
+            + tabbing + "var debugFunctions = PlayFabSettings.LocalDebugFunctions;\n"
+            + tabbing + "var shouldDebugFunction = debugFunctions.Contains(\"*\") || debugFunctions.Contains(request.FunctionName);\n"
```

Comment:
Equals rather than Contains for the first clause. Plus see comments on the Unity SDK as to whether we should do the specific function feature or not.

Reply:
See my comment on https://github.com/nima-ap/SDKGenerator/pull/2

Reply:
Please either explicitly request removal of partial debugging or resolve this to proceed with PR.

Reply:
OK. Please remove partial debugging from this and related PRs. Please put the partial debugging code into a separate branch or branches to allow us to merge it later if we decide to do so.

Reply:
Removed.